src/App.tsx

```diff
@@ -1,7 +1,47 @@
-import React from 'react';
+/* eslint-disable react/sort-comp */
+/* eslint-disable max-len */
```

Comment:
why do we need this?

Reply:
/* eslint-disable react/sort-comp */ - to pass linter errors